Hello Microverse project

Malcom-Charlie opened the pull request:
A review request with regards to the Hello Microverse project

Malcom-Charlie committed:
linters-added

Malcom-Charlie committed:
html added

Malcom-Charlie committed:
css added

Malcom-Charlie committed:
modified linter.yml

Malcom-Charlie committed:
style corrected

Malcom-Charlie committed:
gitignore added

ishimwezachee reviewed (changes requested):
Hi @Malcom-Charlie  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
![](https://camo.githubusercontent.com/6ac905aa3a075ef45de55758c20c6e6efb1dfdbbb0d9fd03d4718d26c5c96cde/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966)

### To highlight 💯 
- Good commit message ✔️ 
- No Linter errors ✔️ 
-  Project is running successfully  ✔️ 


## Required Changes ♻️

- [ ]  You've done well to document your readme..MD file, but its better to improve it because there is some important parts that you've deleted, and they are very important to project documentation. some of them are like, adding an Author to the project, etc. 
Take a reference to this [template ](https://github.com/microverseinc/readme-template)and customize it based on you project. 

Maybe you can take this example,  I guess it is going to help you to know what you need to improve in your Readme file. 

![image](https://user-images.githubusercontent.com/43877689/170143220-1628f058-107f-4b4b-bfe5-94a67b281d1c.png)


- [ ]  You  Pull request does not have a detailed description. You should add a very good description and title of your pull request so that someone will have an idea of what you've worked on before he/she starts reviewing your project. 


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on @Ishimwe Zachee in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
(Not automatically expanded because 3 MB is too large. You can [expand it anyway](https://camo.githubusercontent.com/6ac905aa3a075ef45de55758c20c6e6efb1dfdbbb0d9fd03d4718d26c5c96cde/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966) or [open it in a new window](https://camo.githubusercontent.com/6ac905aa3a075ef45de55758c20c6e6efb1dfdbbb0d9fd03d4718d26c5c96cde/68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f4a713832345239334a734c775a436169534c2f67697068792e676966).)

Malcom-Charlie committed:
Update README.md

tresorsawasawa commented on style.css:
```diff
@@ -0,0 +1,3 @@
+h1 {
+  font-size: 50px;
+}
```
- [ **OPTIONAL** ] I think it would be better to style your heading by using a **class** as required for the project. This should be pointed out by the previous reviewer. I know you can do it :muscle:.

Malcom-Charlie replied:
Thank you [@tresorsawasawa](https://github.com/tresorsawasawa) I will check on that

tresorsawasawa commented on index.html:
```diff
@@ -0,0 +1,14 @@
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My First Project</title>
+    <link rel="stylesheet" href="style.css">
+
```
- [ **OPTIONAL** ] I suggest you remove this empty line to make your code clean and smart.

Malcom-Charlie replied:
I will also check on this. [@tresorsawasawa](https://github.com/tresorsawasawa) thank you.

tresorsawasawa reviewed (changes requested):
# STATUS: REQUEST CHANGES :recycle::recycle: 

Hi @Malcom-Charlie 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

![almost-there](https://user-images.githubusercontent.com/57562869/143511358-9cde7daf-4ad0-4db2-8f15-18b54288ff91.gif)

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 

## Required Changes
- [ ] Kindly make sure your pull request has a good description of every detail you made in the project to make it professional. You can stick for example the [project's requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md#:~:text=documented%20your%20work.-,Requirements,Open%20a%20pull%20request.,-Code%20review) as they are tasks you completed in the project
- [ ] **Check the comments under the review**..

## Optional suggestions
- [ ] Good job so far with some changes you update in the README.md file. So, noticed that they are remaining some other important things to add as said by the previous reviewer and he didn't give precisions for everything to be added. I also suggest you first **copy-paste** the microverse README [template](https://raw.githubusercontent.com/microverseinc/readme-template/master/README.md), edit it according to your project's content by section then leave the **issues** section as it is important in any project to create issues and for the license section you can copy the MIT.md file license from the same [template](https://github.com/microverseinc/readme-template) then edit it.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Malcom-Charlie committed:
applied PR review changes

removed a blank line after the <link> tag as directed by code reviewer [@tresorsawasawa](https://github.com/tresorsawasawa)

Malcom-Charlie committed:
applied PR review changes

changed the style option from font size to background color

Kossi-stack reviewed (approved):
## Status: Approved ✔️
Hi @Malcom-Charlie ,

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 🎉

![](https://user-images.githubusercontent.com/74506933/148243736-d9f63735-4cb1-411f-be28-34972618852d.gif)
You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## Optional suggestions
- [ ] kindly use **class** when you are styling your heading stated by the previous reviewer and aslo don't use color name when you are styling.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
